api/cart/index.js

```diff
@@ -61,6 +61,32 @@
     });
   });
 
+  // Update item
+  app.put("/cart/:id", function (req, res, next) {
+    if (req.params.id == null) {
+      return next(new Error("Must pass id of item to update"), 400);
+    }
+
+     console.log("Updating item: " + req.params.id + " quantity: " + req.body.quantity);
+
+    var custId = helpers.getCustomerId(req, app.get("env"));
+    var options = {
+      uri: endpoints.cartsUrl + "/" + custId + "/items",
+      method: 'PATCH',
```

Comment:
It's implemented as a PATCH in the cart service, and that makes sense actually. PATCH is used usually when updating an item.